src/test/regress/expected/multi_explain_0.out

```diff
@@ -0,0 +1,334 @@
+--
```

Comment:
I think we should disable psql's alignment in this file. Otherwise tiny changes in explain output will result in large output changes. Something like
```
-- temporarily disable fancy output, so output changes create less diff noise
\a\t
$test
-- restore normal output.
\a\t
```
Although I guess we can just skip enabling it again, and use it for the whole multi_explain.sql